### PR TITLE
libobs: Copy both converted textures on stall

### PR DESCRIPTION
**Pull request by kkartaltepe**
### Description

As the comment mentions this only occurs during an encoder stall on the first frame. With opengl texture encoders we might have multiple planes and need to copy both of them.

### Motivation and Context

On very low power devices this happens once in a while, and it consistently happens on a very terrible egpu configuration.

### How Has This Been Tested?

Fixes the first frame only containing the Luma plane on the egpu setup where I can trigger this consistently.

### Types of changes

 - Bug fix (non-breaking change which fixes an issue) 

### Checklist:

- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

**Comment by lextra2** [comment]
`On other platforms these are to distinct textures`

`to` should say `two`

File: libobs/obs-video.c [hunk 1]
```diff
@@ -516,6 +516,13 @@ static inline bool queue_frame(struct obs_core_video_mix *video,
 	 * will ensure better performance. */
 	if (raw_active || vframe_info->count > 1) {
 		gs_copy_texture(tf.tex, video->convert_textures_encode[0]);
+#ifndef _WIN32
+		// y and uv textures are views of the same texture on d3d, and
+		// gs_copy_texture will copy all views of the underlying
+		// texture. On other platforms these are to distinct textures
+		// that must be copied separately.
```
Comment (RytoEX) [inline libobs/obs-video.c:523]:
```suggestion
		// Y and UV textures are views of the same texture on D3D, and
		// gs_copy_texture will copy all views of the underlying
		// texture. On other platforms, these are two distinct textures
		// that must be copied separately.
```

**Review by RytoEX — CHANGES_REQUESTED** [review]
Commit message nits:
* d3d -> D3D
* opengl -> OpenGL

**Review by derrod — APPROVED** [review]

**Comment by kkartaltepe** [comment]
Typos fixed

**Review by RytoEX — APPROVED** [review]
Nits fixed. Leaving to others to finish review/merge.

**Pull request merged**